src/containers/DebitCard/bridgeCard/cardRevealAuth.tsx

```
@@ -24,11 +23,14 @@ import {
   useNavigation,
   useRoute,
 } from '@react-navigation/native';
+import { t } from 'i18next';
+import { Card } from '../../../models/card.model';
+import { capitalize } from 'lodash';
 
 interface RouteParams {
   onSuccess: (data: any, provider: CardProviders) => {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)